package.json

```diff
@@ -12,7 +12,8 @@
         "start": "react-scripts start",
         "build": "react-scripts build",
         "predeploy": "npm run build",
-        "deploy": "gh-pages -d build"
+        "deploy": "gh-pages -d build",
+        "analyze": "source-map-explorer build/static/js/main.*"
```

Comment:
Did you still want to include this? The commit message sounded like you might want to remove it before merging

Reply:
Oh yeah, those were for a different thing I was doing. Don't know how they got into this branch 😐